lib/releaselib.sh

```diff
@@ -910,6 +910,8 @@ release::docker::release () {
     "kube-proxy"
     "hyperkube"
   )
+  # ml_platforms should be in the os1/arch1,os2/arch2,os3/arch3 form
```

Comment:
what does `ml` stand for? I keep reading "machine learning", which I'm sure is wrong.